sdk/src/vote_program.rs

```diff
@@ -53,29 +80,32 @@ pub enum VoteInstruction {
 
 #[derive(Debug, Default, Serialize, Deserialize, PartialEq, Eq)]
 pub struct VoteState {
-    pub votes: VecDeque<Vote>,
+    pub votes: VecDeque<StoredVote>,
     pub node_id: Pubkey,
     pub staker_id: Pubkey,
+    pub max_lockout_slot_height: u64,
```

Comment:
starts out at zero?

Reply:
maybe start moving toward "block" names, instead of "slot", probably should be an Option?

Reply:
"slot" looks like the appropriate term to me.  When we're talking about time, it should be in ticks, hashes, epochs.